### PR TITLE
Convert prometheus bridge readme to godocs

### PR DESCRIPTION
**Pull request by dashpole**
fixes https://github.com/open-telemetry/opentelemetry-go-contrib/issues/4339

### CODE REVIEW

**Comment by dashpole** [comment]
@pellared I wanted to make the example print something out to the console, but that requires importing the stdoutmetric package.  Is there a way to add dependencies only for the example?

**Comment by pellared** [comment]
> @pellared I wanted to make the example print something out to the console, but that requires importing the stdoutmetric package. Is there a way to add dependencies only for the example?

No 😢 We had the same problem in the sdk/metric example. We decided to simply use a ManualReader instead. See https://github.com/open-telemetry/opentelemetry-go/blob/9bbefc6cc31995c7bdd37c69eb153fc2201c3562/sdk/metric/example_test.go#L47-L50

**Review by MrAlias — APPROVED** [review]

File: bridges/prometheus/example_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+package prometheus_test
+
+import (
+	"go.opentelemetry.io/contrib/bridges/prometheus"
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+func ExampleNewMetricProducer() {
+	// Create a Promethes bridge "Metric Producer" which adds metrics from the
+	// [prometheus.DefaultGatherer]. Add the [WithGatherer] option to add metrics
```
Comment (pellared) [inline bridges/prometheus/example_test.go:24]:
```suggestion
	// prometheus.DefaultGatherer. Add the WithGatherer option to add metrics
```

These are just regular code comments. The "Go Doc" syntax does not work here.

**Review by pellared — APPROVED** [review]
One suggestion. Awaiting for a non-Splunk approval.

**Comment by pellared** [comment]
Should we also add "Bridges" here: https://github.com/open-telemetry/opentelemetry-go-contrib#contents?

**Comment by dashpole** [comment]
> Should we also add "Bridges" here: https://github.com/open-telemetry/opentelemetry-go-contrib#contents?

Done https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4348/commits/35c2be67700cc0cde69626415c57de704b03e7f0

**Review by pellared — APPROVED** [review]

**Review by hanyuancheung — APPROVED** [review]

**Pull request merged**